assets/queries/cloudFormation/aws/s3_bucket_acl_allows_read_or_write_to_all_users/metadata.json

```diff
@@ -3,7 +3,7 @@
   "queryName": "S3 Bucket ACL Allows Read Or Write to All Users",
   "severity": "HIGH",
   "category": "Access Control",
-  "descriptionText": "S3 Buckets sould not be readable and writable to all users",
+  "descriptionText": "S3 Buckets should not be readable and writable to all users",
```

Comment:
The same typo occurs in `assets/queries/cloudFormation/aws/s3_bucket_acl_allows_read_to_all_users/metadata.json`. Can you also fix it, please? Thank you so much in advance!